llvm/docs/HowToCrossCompileLLVM.rst

```diff
@@ -6,210 +6,202 @@ Introduction
 ============
 
 This document contains information about building LLVM and
-Clang on host machine, targeting another platform.
+Clang on a host machine, targeting another platform.
 
 For more information on how to use Clang as a cross-compiler,
 please check https://clang.llvm.org/docs/CrossCompilation.html.
 
-TODO: Add MIPS and other platforms to this document.
+This document describes cross-building a compiler in a single stage, using an
+existing ``clang`` install as the host compiler.
 
-Cross-Compiling from x86_64 to ARM
-==================================
 
-In this use case, we'll be using CMake and Ninja, on a Debian-based Linux
-system, cross-compiling from an x86_64 host (most Intel and AMD chips
-nowadays) to a hard-float ARM target (most ARM targets nowadays).
+Cross-Compiling from x86_64 to ARM, AArch64, or RISC-V
+======================================================
 
-The packages you'll need are:
+These instructions have been tested for targeting 32-bit ARM, AArch64, or
+RISC-V but should be equally applicable to any other target.
```

Comment:
Could put this in a `.. note` so it stands out, because someone *will* miss it.

Maybe I'll think different when I've reviewed the whole thing but putting the arch names in the subtitle is probably best overall just for search purposes.

Reply:
I've done this and changed the heading, as the overall nesting of headings didn't make sense anyway.